riva/proto/riva_tts.proto

```diff
@@ -39,6 +39,18 @@ message RivaSynthesisConfigResponse {
     repeated Config model_config = 1;
 }
 
+// Required for PFlow model
+message ZeroShotData {
+    // Small (3-sec) audio prompt for PFlow model.
+    bytes audio_prompt = 1;
+    // Sample rate for input audio prompt. Current defaults to 22050, can be user provided.
+    int32 sample_rate_hz = 2;
+    // Encoding of audio prompt, defaults to PCM.
+    AudioEncoding encoding = 3;
+    // The number of times user wants to pass audio to resampler. Current defaults to 20.
+    int32 quality = 4;
+}
```

Comment:
will be good to specify a max threshold value that user can set for audio_prompt and quality fields?

Reply:
Updated comments for both.